Musicplayer.css

```diff
@@ -109,6 +121,25 @@ ul li ul li:hover{background: black}
 	text-align:center;
 }
 
+.section3{
```

Comment:
21/9: This is the div box that hold the function that calculate the total duration of all the song that display below.